### PR TITLE
Feature/ansible k8s

### PR DESCRIPTION
**Pull request by mateuszpierzchala-splunk**

### CODE REVIEW

File: ansible/tasks/docker_swarm/deploy_stack.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+- name: Copying docker-compose.yml file on the server
+  copy:
+    src: "/opt/ansible/app/docker-compose.yml"
+    dest: "/home/{{ ansible_user }}/docker-compose.yml"
+    owner: "{{ ansible_user }}"
+    group: "{{ ansible_user }}"
+    mode: u=rw,g=rw,o=r
+
+- name: Create a directory if it does not exist
```
Comment (rjha-splunk) [inline ansible/tasks/docker_swarm/deploy_stack.yml:10]: - name: Create sc4s dependency directories 
  file:
    path: "{{ item }}"
    state: directory
    mode: 0755
  loop:
    - /opt/sc4s
    - /opt/sc4s/tls
    - /opt/sc4s/archive
    - /opt/sc4s/local

**Review by rjha-splunk — COMMENTED** [review]
Added the comment, kindly review and merge.

File: ansible/tasks/docker_swarm/deploy_stack.yml (hunk 1 above)
Comment (mateuszpierzchala-splunk) [reply ansible/tasks/docker_swarm/deploy_stack.yml]: fixed

**Pull request merged**